docs/docs/installation/index.md

```diff
@@ -1,44 +0,0 @@
-# Install Mathesar
```

Comment:
I find this page to be much better as it lets me skim through the available installation options. Any reason why you are deleting it?

Reply:
I moved the list of installation options to the homepage.